### PR TITLE
Preventing changes that break backwards compatibility

### PR DESCRIPTION
**Pull request by ronykoz**
## Status
Ready

## Related Issues
relates to https://github.com/demisto/etc/issues/14407

## Description
sections 1-3 in the issue

### CODE REVIEW

**Comment by ronykoz** [comment]
Can you think of any reason we want to have an option to disable this checks in CircleCI?

File: Tests/scripts/validate_files_structure.py [hunk 1]
```diff
@@ -327,6 +327,26 @@ def get_to_version(file_path):
         return data_dictionary.get('toversion', 'current')
 
 
+def changed_command_name_or_arg(file_path):
+    change_string = run_git_command("git diff HEAD {0}".format(file_path))
+    search_groups = re.search("-([ ]+)?- name: .*", change_string)
+    if search_groups:
+        print_error("You've changed the name of a command or its arg in the file {0} please undo, the line is:\n{1}".format(file_path, search_groups.group(0)[1:]))
```
Comment (anara123) [inline Tests/scripts/validate_files_structure.py]: Possible backwards compatibility break

File: Tests/scripts/validate_files_structure.py [hunk 2]
```diff
@@ -327,6 +327,26 @@ def get_to_version(file_path):
...
+        return True
+
+    return False
+
+
+def changed_context(file_path):
+    change_string = run_git_command("git diff HEAD {0}".format(file_path))
+    search_groups = re.search("-([ ]+)?- contextPath: .*", change_string)
+    if search_groups:
+        print_error("You've changed the context in the file {0} please undo, the line is:\n{1}".format(file_path, search_groups.group(0)[1:]))
```
Comment (anara123) [inline Tests/scripts/validate_files_structure.py]: same here

**Comment by ronykoz** [comment]
@anara123 updated the error messages and added support in moving those lines in the yml file

**Pull request merged**